build/inspr-stack/README.md

```diff
@@ -0,0 +1,99 @@
+# Inspr Stack helm chart
+
+This chart creates a Inspr Stack deployment on a Kubernetes cluster using the Helm package manager, which consist of installing both Inspr Daemon and UIDP at the same time.
+
+
+## Prerequisites
+---
+- Kubernetes 1.16+
+- Helm 3+
+
+## Get Repo Info
+---
+If you’re installing the Inspr chart via Helm, first you need to add the Inspr repository with the command:
+
+```
+$ helm repo add inspr https://inspr-charts.storage.googleapis.com/
+```
+Then, make sure everything is up-to-date:
+
+```
+$ helm repo update
+```
+
+And finaly to see the details from Inspr on the helm you can do:
+
+```
+$ helm search repo inspr-stack
+
+NAME                    CHART VERSION   APP VERSION     DESCRIPTION                                       
+inspr/inspr-stack       v0.1.3          v0.1.3          Helm chart for installing Inspr's Deamon applic...
+```
+
+## Install Chart
+---
+
+```
+$ helm install <your.release.name> inspr/inspr-stack --set insprd.apps.createNamespace=true
```

Comment:
Also I think you can explain the create namespace value. Just like:
"Notice that the value createNamespace is set to true so the .... (continue)"